benchmark/parse_output_gi.py

```diff
@@ -0,0 +1,79 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+#
+# Parse output files of GI tools.
+#
+
+import sys
+import click
+from skbio import Sequence
+
+
+def parse_output_gi(genbank_fp,
+                    gi_fp):
+    """ Identify genes contained in GIs
+
+    Parameters
+    ----------
+    genbank_fp: string
+        file path to genome in GenBank format
+    gi_fp: string
+        file path to GI list
+
+    Returns
+    -------
+    output: string
+        gene names (protein_ids) separated by newline
+    """
+    genes = {}
+    gb = Sequence.read(genbank_fp, format='genbank')
+    for feature in gb.interval_metadata.features:
+        if feature['type_'] == 'CDS':
+            if 'protein_id' in feature:
+                protein_id = feature['protein_id'].replace('\"', '')
+                loc = gb.interval_metadata.features[feature]
+                start = loc[0][0] + 1
+                end = loc[0][1]
+                if protein_id not in genes:
```

Comment:
Good point! I moved this test 3 lines upper. I think CDS.protein_id should be unique in a valid GenBank file. Plus, it has to be unique in our analysis. This test is to avoid some unexpected cases.